### DISCUSSION TITLE
Capturing values from text with colors through #action ?

### DISCUSSION THREAD

**Discussion by amelsen** [scandum/tintin#126] [Script help]
So.. I have a prompt that looks like this, with green colors.. I can't seem to capture the values in there. How do I do that?

![image](https://user-images.githubusercontent.com/3955258/191944425-1da3e056-2453-4e4b-93e2-d840e89ce532.png)

**Comment by scandum** [comment]
Should be as simple as:
```
#act {V: %1 H: %2 S: %3]} {#line ignore #showme V(%1) H(%2) S(%3)}
```
You can use:
```
#line debug #showme V: 0  H: 94%  S: 100%]
```
To see what happens with additional debug info

**Comment by amelsen** [comment]
Action doesn't respond or capture anything at all... as my only action.. it just doesn't want to capture anything?

**Comment by scandum** [comment]
The example I gave works for me.

It's possible your mud does something unusual, but actions are generally pretty easy to trigger.

**Reply by amelsen** [reply]
Yeah.. I have no problems with other actions... but for some odd reason, this isnt capturing it.

Is there a way to set up actions so it searches for specific content, regardless of the amount of spaces/tabs in between?

**Comment by scandum** [comment]
You can create less specific actions, like for example:
```
#act {V:%1H:%2S:%3]} {#line ignore #showme <118>Result: (%1,%2,%3)}
```

**Reply by amelsen** [reply]
I still cant make this work at all... it just doesn't catch anything. Is it because there is '%' in the prompt text?

'V: 12  H: 64%  S: 68%  608158 ]'

So what I would like for the action to catch here is:
0
64
68
608158

I THINK there might be a chance that the reason it doesnt work is because of the '%' in the text? How do I ignore a character when setting up an action? Doesn't seem apparent to me by reading #help pcre

thanks.

**Reply by scandum** [reply]
The % shouldn't matter. Try using `#debug action on` and generate your prompt, it's possible another action is triggering on that line. Only one action can trigger per line.

If that doesn't do the trick see if you can make a more generic action to trigger that line, and then increase complexity so you know where things go wrong.

**Reply by amelsen** [reply]
Ohhh THAT was the issue... I had an old action also! Totally missed that... thanks =) Now it works :)
